m4_sessions/1903_inning/automation.md

```diff
@@ -1,53 +1,48 @@
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission at Turing is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
-## Pre-Session
+#### Introductions 1:00 - 1:05
+  * Have students introduce themselves
+    Name:
+    CoHort: 
+    Pronouns:
+    Answer Questions:
+   1. If you could have one process in your life automated tomorrow what would it be?
+   2. What automated process today do you appreciate the most?
 
+#### Icebreaker 1:05 - 1:10
+   Automation icebreaker?
 
-### Survey Questions
-Survey to be sent either prior to session or immediately after ( prior may lead to better response? )
+### Groupings 1:10 - 1:15
+  We will group members by opposing stances. Have students stand in a spectrum of how much they like the idea of automation, and from that hand select one each from opposing sides and a middle. This should facilitate the most conversation and opposing viewpoints.
 
-#### Survey to lighten the mood
-  * funny questions like:
-   1. When the robots attack, what’s the first thing you do?
-   2. If you could have one process in your life automated tomorrow what would it be?
-   3. What automated process today do you appreciate the most
-   4. Write your apology to the person whose job you stole by automating that thing.
-   
-### Pre-Reading
+#### Whiteboard Pros and Cons 1:15 - 1:25
+  Have groups write pros and cons on board or papers.
+  Each group shares a pro and con  
+  
+### POM 1:25
 
-Send pre-reading out at least 24 hours before the session
+#### Watch Video 1:30 - 1:45
+ 
+#### Answer Questions In Notebook 1:45 - 1:50
 
-*   ["Machines for Thinking"](http://www.economist.com/news/books-and-arts/21669597-computers-will-get-smarter-humans-charge-machines-thinking) (_The Economist_: October 1, 2015)
-*   "The Rise of the Robots by Martin Ford / Humans Need Not Apply by Jerry Kaplan – review"
-https://www.theguardian.com/books/2015/oct/01/the-rise-of-robots-humans-need-not-apply-review (_The Guardian_: October 1, 2015)
-*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+1) How serious an impact do you think automation will have on our economy from now until you retire? <br> 
+2) Where do you think you fall across the spectrum of people that will be affected by these changes? <br> 
+3) What do you think it means to be a responsible citizen around this issue? <br> 
+4) What role can Turing play in advancing civic responsibility in response to the challenges automation will bring?
```

Comment:
What role do you have as a software developer?  What is the danger of not considering these questions?

Reply:
We will add this to the list of questions.